mail/templates/email_base.html

```diff
@@ -239,7 +239,7 @@
 	            <tr>
 	                <td style="padding: 20px;">
                       {% block logo %}
-                      <img src="{{ base_url }}/static/images/mit-logo.png" width="160" height="80" alt="{{ site_name }}" style="height: auto; background: #ffffff; font-family: sans-serif; font-size: 15px; line-height: 15px; color: #A31F34; margin-bottom: -10px;  margin-right: 10px;"/>
+                      <img src="{{ base_url }}/static/images/mit.png" width="160" height="80" alt="{{ site_name }}" style="height: auto; background: #ffffff; font-family: sans-serif; font-size: 15px; line-height: 15px; color: #A31F34; margin-bottom: -10px;  margin-right: 10px;"/>
```

Comment:
I think we can remove font-related styles from this tag?

```suggestion
                      <img src="{{ base_url }}/static/images/mit.png" width="160" height="80" alt="{{ site_name }}" style="height: auto; margin-bottom: -10px;  margin-right: 10px;"/>
```

Reply:
resolved